packages/neuron-ui/src/utils/hooks/index.ts

```diff
@@ -405,10 +405,22 @@ export const useToggleChoiceGroupBorder = (containerSelector: string, borderClas
     }
   }, [containerSelector, borderClassName])
 
-export const useGlobalNotifications = (
-  dispatch: React.Dispatch<{ type: AppActions.SetGlobalDialog; payload: State.GlobalDialogType }>,
-  hasDismissMigrate: boolean
-) => {
+export const useMigrate = () => {
+  const [isMigrateDialogShow, setIsMigrateDialogShow] = useState(false)
+  const [hasDismissMigrate, setHasDismissMigrate] = useState(false)
+  const [ckbDataPath, setCkbDataPath] = useState<string>()
+  useEffect(() => {
+    getCkbNodeDataPath().then(res => {
+      if (isSuccessResponse(res) && res.result) {
+        setCkbDataPath(res.result)
+      }
+    })
+  })
```

Comment:
This doesn't seem to be something that should be executed on every render, did you forget to write deps?

Reply:
You are right, I miss the deps. It will add the dep when location. path changed.